perk-store-webapps/src/main/webapp/css/main.less

```diff
@@ -1176,9 +1196,41 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
         }
       }
     }
+    .product-list {
```

Comment:
Could we avoid duplicating those styles ?

Reply:
for each media size we need a different style